src/antsibull/pypi.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (C) 2024 Maxwell G <maxwell@gtmx.me>
+# SPDX-License-Identifier: GPL-3.0-or-later
+# GNU General Public License v3.0+
+# (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+Work with the PyPI API
+"""
+
+import datetime
+from pathlib import Path
+from typing import NamedTuple, Optional
+from urllib.parse import urljoin
+
+import aiohttp
+import pydantic as p
+from antsibull_core.utils.hashing import verify_hash
+
+PYPI_BASE_URL = "https://pypi.org/pypi/"
+
+
+class SdistAndWheel(NamedTuple):
```

Comment:
I think it's useful to mention that this encapsulates both values simultaneously.
```suggestion
class SdistAndWheelPair(NamedTuple):
```

Reply:
Good point.